### PR TITLE
 SBAELIDEV-324 adding front end functionality for viewing discussions on a learning event

### PR DESCRIPTION
**Pull request by kdyer**

### CODE REVIEW

File: front-end/src/components/Discussion/LearningEventDiscussion.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+import "./LearningEventDiscussion.scss";
+
+export default class LearningEventDiscussion extends Component {
+
+  commentCount(count) {
+    if (count > 0) {
+      return <span className="comment-count">({count})</span>
+    }
+    return ""
```
Comment (mrcastle) [inline front-end/src/components/Discussion/LearningEventDiscussion.js]: I believe convention should be to return null here

https://reactjs.org/docs/jsx-in-depth.html#booleans-null-and-undefined-are-ignored

Possible other method to render your count here in docs here.

File: front-end/src/components/Discussion/UserComment.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    }
+    return "http://picsum.photos/45"
+  }
+
+  render() {
+    var reply = this.props.replies
+
+    return <Grid.Row className='user-comment' width={16}>
+      <Grid.Column width={1}>
+        <img className='user-image' src={this.userImage(reply.user_img)} alt={reply.user_name}/>
```
Comment (mrcastle) [inline front-end/src/components/Discussion/UserComment.js]: Semantic image component could be used here

File: front-end/src/components/Discussion/UserComment.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
+import React, {Component} from "react";
+import {Grid} from "semantic-ui-react";
+import Discussion from "./Discussion";
+
+export default class UserComment extends Component {
+
+  replyLink(parentContentType, parentId, replies) {
+    if (parentContentType !== "comment" && replies && replies.replies) {
+      return <Grid.Row>
+        <a className="reply-link" href={`#/reply?postId=${parentId}`}>Reply</a>
```
Comment (mrcastle) [inline front-end/src/components/Discussion/UserComment.js]: `<Link>` component for here?

File: front-end/src/components/Discussion/DiscussionPost.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+              <Form.Button primary id='submit_post' onClick={this.submitPost}>
+                Add Comment
+              </Form.Button>
+            </Form.Group>
+            <Input type="hidden" name="content_id" value={this.state.parentId}/>
+          </Form>
+        </Grid.Column>
+      </Grid.Row>
+    }
+    return ''
```
Comment (mrcastle) [inline front-end/src/components/Discussion/DiscussionPost.js]: Can return null here

File: front-end/src/components/Discussion/UserComment.js (hunk 3 above)
Comment (kdyer) [reply front-end/src/components/Discussion/UserComment.js]: I thought yes but this isn't going to link to a page/content, it's going to ajax in a reply box for users to comment

**Pull request merged**